### PR TITLE
[chores] Droped support for Django 2.2 #135

### LINKED ISSUE #135: [deps] Drop support for Django 2.2
Drop support for Django 2.2 and remove any compatibility shims for that version in the repository.

### PR DESCRIPTION
**Pull request by niteshsinha17**
I have made changes. If anything is left please let me know I will be very happy to solve it.
Fixes #135

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -5,3 +5,4 @@ openwisp-utils[rest]~=0.6
 swapper~=1.1
 markdown>=3.2,<3.3
 celery~=4.4
+django>=3.0,<3.2
```
Comment (pandafy) [inline requirements.txt:8]: I don't think you will need to add this here.

**Review by pandafy — CHANGES_REQUESTED** [review]
@niteshsinha17  can you fix the commit message. Change the tag to `chores` and drop the trailing bracket (`]`)

File: requirements.txt (hunk 1 above)
Comment (niteshsinha17) [reply requirements.txt:8]: Okay I will do the changes.

**Review by pandafy — DISMISSED** [review]
LGTM! Thanks for your patience @niteshsinha17 :smile:

**Review by pandafy — CHANGES_REQUESTED** [review]
@niteshsinha17 can you rebase this PR on current master and make changes according to GitHub Actions. This should be quick.

**Comment by niteshsinha17** [comment]
@pandafy  Okay, I will try to do it today.

**Pull request closed without merging**